helpers/controllers/alertController.ts

```diff
@@ -16,7 +16,11 @@ export const addAlert = async (
   ctx: { req: LoggedRequest }
 ) => {
   const { req } = ctx
+  const isAdmin = _.get(req, 'user.isAdmin', false)
```

Comment:
could you create a helper function for this? In the future we are going to change away from strings, so if you create a helper function then in the future when we change to `boolean` we only need to change in 1 place instead of everywhere.

```
if (!isAdmin(req)) {
  throw new Error('User is not admin')
}
```

And your `isAdmin` function should check for both string and boolean at the moment:

```
const isAdmin = (req) => {
  return _.get(req, 'user.isAdmin', false) === 'true'
}
```